packages/fluentui/react-builder/src/components/Canvas.tsx

```diff
@@ -264,82 +268,92 @@ export const Canvas: React.FunctionComponent<CanvasProps> = ({
       id={iframeId}
     >
       <FrameContextConsumer>
-        {({ document, window }) => (
-          <>
-            {(!jsonTree.props?.children || jsonTree.props.children.length === 0) && (
-              <div
-                style={{
-                  padding: '8rem',
-                  textAlign: 'center',
-                  position: 'absolute',
-                  pointerEvents: 'none',
-                  width: '100%',
+        {({ document, window }) => {
+          const jss = create({
+            plugins: [...jssPreset().plugins],
+            insertionPoint: document.head,
+          });
+          return (
+            <>
+              {(!jsonTree.props?.children || jsonTree.props.children.length === 0) && (
+                <div
+                  style={{
+                    padding: '8rem',
+                    textAlign: 'center',
+                    position: 'absolute',
+                    pointerEvents: 'none',
+                    width: '100%',
+                  }}
+                >
+                  <span style={{ fontSize: '4rem' }} role="img" aria-label="Finger pointing left">
+                    👈
+                  </span>
+                  <div style={{ fontSize: '1.2rem', opacity: 0.5 }}>Drag n' Drop some components</div>
+                </div>
+              )}
+
+              <DebugSelector
+                active={isSelecting}
+                key={`debug-selector-${selectedComponent?.uuid ?? 'unknown'}`}
```

Comment:
Can it happen it will have more than 1 `key="'unknown'"`? if so we should probably generate something random here.